src/content/en/2021/structured-data.md

```diff
@@ -17,4 +19,568 @@ featured_stat_3: TODO
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3kZ1Ys-9tId-WBa_8muMzrTAu1Ad5TXYgkopXMmBVc1xmd2N_4PZIpEZEOqRL7baymle0kHzaC6KY/pubchart?oid=501587417&format=interactive",
+  sheets_gid="1876349096"
+  )
+}}
+
+We can also see that _RDFa_ and _Open Graph tags_ in particular are extremely prevalent, appearing on 60.61% and 57.45% of pages respectively.
+
+At the other end of the scale, legacy formats, like _Microformats_ and _microformats2_, appear on fewer than 1% of pages.
+
+### Coverage by syntax type
+
+In addition to identifying when a certain type of structured data is present, we collect information on the types of data it describes. We can break each of these down and explore how each format and syntax is being used.
+
+#### RDFa
+
+<a hreflang="en" href="https://www.w3.org/TR/rdfa-lite/">Resource Description Framework in Attributes</a> (RDFa) is a technology for linked data markup, which was introduced by W3C in 2015. It allows users to augment and translate visual information on a web page by adding additional attributes to markup.
+
+For example, a website owner might add a `rel="license"` attribute to a hyperlink in order to explicitly describe it as a link to a licensing information page.
+
+{{ figure_markup(
+  image="structured-data-rdfa-types.jpg",
+  caption="RDFa types",
+  description="Bar chart showing the usage of RDFa across the web. Of all mobile pages in our data: `foaf:image` is on 0.86%, `foaf:document` is on 0.36%, `sioc:item` is on 0.24%, `schema:webpage` is on 0.11%, `image` is on 0.9%, `listitem` is on 0.08%, `breadcrumblist` is on 0.07%, `og:website` is on 0.06%, `skos:concept` is on 0.04%, `webpage` is on 0.04%, `v:breadcrumb` is on 0.04%, `schema:article` is on 0.03%, `sioc:useraccount` is on 0.03%, and `person` is on 0.03%. Desktop usage is similar.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3kZ1Ys-9tId-WBa_8muMzrTAu1Ad5TXYgkopXMmBVc1xmd2N_4PZIpEZEOqRL7baymle0kHzaC6KY/pubchart?oid=1063480738&format=interactive",
+  sheets_gid="379443022",
+  width=600,
+  height=504,
+  sql_file="rdfa_type_ofs.sql"
+  )
+}}
+
+When we evaluate the types of RDFa, we can see that the `foaf:image` syntax is present on far more pages than any other type—on upwards of 0.86% of all pages in our data set. Whilst that may seem like a small proportion, it represents over ~65,000 pages, and over 60% of the total RDFa markup that we discovered.
+
+Beyond this outlier, the use of RDFa diminishes and fragments considerably, though there are still some interesting discoveries to explore.
+
+##### On FOAF
+
+[FOAF](http://xmlns.com/foaf/spec/) is a linked data dictionary of people-related terms, created in the early-2000s. It can be used to describing _people_, _groups_ and _documents_.
```

Comment:
Does FOAF stand for something? Should we mention that?

Reply:
Clarified.